$NetBSD: patch-ak,v 1.4 2006/06/11 15:05:39 wiz Exp $

Needed to have a NULL macro in the program that follows these includes.

--- configure.orig	2006-05-29 13:25:13.000000000 +0000
+++ configure
@@ -26341,6 +26341,7 @@ cat >>conftest.$ac_ext <<_ACEOF
 		#include <sys/socket.h>
 		#include <netinet/in.h>
 		#include <netdb.h>
+		#include <stddef.h>
 
 
 int
